### ISSUE TITLE
Showing accessibility error 'ARIA attribute value is incorrect' on 'Search and filter products and services' page.

### ISSUE THREAD

**Issue by Shalini-git25** [DFE-Digital/find-information-products-services-frontend#24]
**Describe the bug**
On 'Search and filter products and services' page, ran tests using 'ARC toolkit' to test accessibility issues. Showing accessibility error as - "ARIA attribute value is incorrect".

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Search and filter products and services' page.
2. Open ARC Toolkit in dev tools.
3. Select 'Run tests' to start the automated accessibility checks.
4. In Result section error message is displayed as 'ARIA attribute value is incorrect'.
5. Scroll down on the page - error is displayed on pagination link no 1.

**Expected behavior**
No accessibility error should be present on page.

**Screenshots**

<img width="2232" height="1322" alt="Image" src="https://github.com/user-attachments/assets/bd38a6a6-e392-4dbe-a82d-9b879dfd9f53" />
